mm-common/PKGBUILD

```diff
@@ -1,26 +1,20 @@
 # Maintainer: Jeremy Drake <github@jdrake.com>
 
 pkgname=mm-common
-pkgver=1.0.2
+pkgver=1.0.3
 pkgrel=1
 pkgdesc="Common build files of the C++ bindings"
 url="https://www.gtkmm.org/"
 arch=('i686' 'x86_64')
-makedepends=('git' 'meson')
+makedepends=('meson')
 license=('GPL2')
-_commit=85381b86dcc2490d2b080c178477df9eb65103ca  # tags/1.0.2^0
-source=("git+https://gitlab.gnome.org/GNOME/mm-common.git#commit=$_commit"
+source=("https://download.gnome.org/sources/mm-common/${pkgver%.*}/${pkgname}-${pkgver}.tar.xz"
```

Comment:
Cool, when I did the initial package I didn't find source tarballs (though to be honest once I found the arch package used git, I didn't look too hard)

Reply:
Arch uses git repos because git is itself a checksum verification engine (kind of). But that comes at a cost, the git repo is cloned with full history.